lxd/fsmonitor/fsmonitor_interface.go

```diff
@@ -1,5 +1,17 @@
 package fsmonitor
 
+const (
+	// DriverNameFANotify is the name of the FANotify driver.
+	//
+	// FANotify should be preferred over INotify because it is more performant and does not need to recursively watch
+	// subdirectories. However, it is not possible to use fanotify if the specified path is not a mountpoint because we
+	// need to use the unix.FAN_MARK_FILESYSTEM flag for this functionality.
+	DriverNameFANotify = "fanotify"
```

Comment:
Do these constants need exporting?

Reply:
I was also thinking this, but I figured that they're constants and exporting the names gives the caller a small amount of information about the underlying implementation. I realise the point of an interface is so that the caller doesn't have to care, but it's nice documentation wise to get a little overview at the top level.